.github/CODEOWNERS

```diff
@@ -13,6 +13,7 @@
 /src/plugins/saved_search/ @elastic/kibana-data-discovery
 /x-pack/plugins/discover_enhanced/ @elastic/kibana-data-discovery
 /test/functional/apps/discover/ @elastic/kibana-data-discovery
+/test/functional/apps/context/ @elastic/kibana-data-discovery
```

Comment:
Oh, we weren't code owners? thx! for adding it. we should move `context` to the `discover` folder in the future. it still separated here, because back in the old days they were different plugins